.github/workflows/integration-test.yml

```
@@ -174,7 +174,7 @@ jobs:
     strategy:
       matrix:
         ARCH: [ "amd64", "arm64" ]
-        PROFILE: [ "integration", "smartagent" ]
+        PROFILE: [ "smartagent" ]
```

```suggestion
        PROFILE: [ "integration", "smartagent" ]
```
TEST: Remove this change once testing is complete.